<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RCC Extractor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_rcc">
          <property name="text">
           <string>RCC File:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="line_edit_rcc">
          <property name="placeholderText">
           <string>Please Select a RCC File.</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="button_browse_rcc">
          <property name="minimumSize">
           <size>
            <width>88</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_exdir">
          <property name="text">
           <string>Extra Dir:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="line_edit_exdir">
          <property name="placeholderText">
           <string>Please Select Extra Dir.</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="button_browse_exdir">
          <property name="minimumSize">
           <size>
            <width>88</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QListView" name="list_view"/>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="button_extra_selected">
            <property name="text">
             <string>Extra Selected</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_extra_all">
            <property name="text">
             <string>Extra All</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="button_exit">
            <property name="text">
             <string>Exit</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>button_browse_rcc</tabstop>
  <tabstop>line_edit_rcc</tabstop>
  <tabstop>button_extra_selected</tabstop>
  <tabstop>button_exit</tabstop>
  <tabstop>list_view</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
